plugins/sbt/sbt-test/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# check if the file gets created
+sbt clean compile graphqlCodegen --debug
+generateCode=target/scala-2.13/src_managed_graphql/io/github/kobylynskyi/graphql/test/model/User.java
+if [ -f "$generateCode" ]; then
```

Comment:
this is unusual. typically the way you'd do this is like https://github.com/kobylynskyi/graphql-java-codegen/blob/4379b02cb3a2f59c6ca854a7ffc38c22e0b61a43/plugins/sbt/src/sbt-test/sbt-graphql-java-codegen/simple/test

Reply:
Sorry, I have never used the similar this "test", because I can't know the file type at all. I can't know how this is triggered. I don't see the use of github workflows

Reply:
It's executed when you run `sbt scripted`, which is the standard way of testing sbt plugins. You can see the docs here: https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html